### DISCUSSION TITLE
Issue: Can not swipe away notification feed items on tablets

### DISCUSSION THREAD

**Discussion by tjsuominen** [community/community#124604] [Mobile] [answered]
How to repeat: Use tablet. Mine is Google Pixel. Choose the notifications feed from main screen. Try to dismiss notifications one by one by sweeping them right. Extremely difficult, they are 'stuck' on screen. 

Works on smaller screens e.g. Pixel 8 Pro, but not on big screens.

**Comment by alcere** [comment]
Hi @tjsuominen! We apologize for the inconvenience! I have tried to reproduce the issue, but everything appears to be working correctly on my end. Could you please take a screenshot or record a video to help us better understand and resolve your problem? Thank you!

**Reply by tjsuominen** [reply]
Will do, give me a couple of days (latest during the weekend).

**Reply by tjsuominen** [reply]
https://github.com/user-attachments/assets/94e80d0b-e29e-4193-9f1d-7243c28f52b0

Finally remembered to record the screen (on Pixel Tablet). This (= basically nothing) is what happens when trying to swipe right ('mark as seen' aka removes the notification item) notification items on tablet. Works perfectly on my Pixel 8 Pro.

Ping @alcere 

HTH

**Reply by alcere** [reply]
Thanks for sharing @tjsuominen. I will open a bug and we will investigate this! thanks for reporting!

**Reply by alcere** [reply]
@tjsuominen one quick question: for the exact same notification, can you execute the swipe gesture on mobile but not using your tablet? Not all actions are available for all notification types, so we would like to know if this is an issue affecting only notification for releases (as in the video you shared). Thanks!

**Reply by tjsuominen** [reply]
Yes. Works 100% on phone.

**Reply by elisealix22** [reply]
@tjsuominen we're having a bit of trouble reproducing this so I included some demo videos on a Pixel Tablet below in case we are doing something different. What Android version is your Pixel Tablet running? Does it happen intermittently or is it consistent for you? Sorry you're having trouble!

<details><summary>Unsubscribe / Mark as done demo</summary>

https://github.com/user-attachments/assets/b7eb2028-efee-4687-b5b3-9d9c9c348974

</details>

<details><summary>Mark as read demo</summary>

https://github.com/user-attachments/assets/2e41281d-7944-4bec-9b14-9d958881e360

</details>

**Reply by tjsuominen** [reply, marked as answer]
Seems that the issue has been vanished and now works as designed 🙈. I have not done anything especial except kept my phone and apps updated. The issue can be closed, sorry for the hassle.

**Reply by alcere** [reply]
We are glad to read that! Thanks for your help reporting this issue @tjsuominen!
